src/back/TicketManager/tests/TicketManager.IntegrationTests/TestBase.cs

```diff
@@ -21,9 +23,10 @@ public class TestBase : IAsyncDisposable
     protected readonly HttpClient AnonymousClient;
     protected readonly HttpClient UserClient;
     protected readonly HttpClient OrganizerClient;
+    protected readonly HttpClient AdminClient;
 
-    protected readonly UserDto DefaultUser;
-    protected readonly OrganizerDto DefaultOrganizer;
+    protected UserDto DefaultUser;
+    protected OrganizerDto DefaultOrganizer;
 
     public TestBase()
```

Comment:
I don't know if you can see this, but GitHub started giving us warnings that were found during build. It would be nice if you could somehow fix it. I know C# doesn't understand setting fields in some method and not in constructor, but maybe there is a sollution
![image](https://user-images.githubusercontent.com/43820607/229837900-5162a45e-502e-4a30-b64e-747bfa41d0ab.png)